census/README.md

```diff
@@ -323,41 +332,47 @@ gsutil ls -r $GCS_JOB_DIR/export
 
 
  * Estimator Based: You should see a directory named `$GCS_JOB_DIR/export/census/<timestamp>`.
- ```
+```
  export MODEL_BINARIES=$GCS_JOB_DIR/export/census/<timestamp>
- ```
+```
 
  * Low Level Based: You should see a directory named `$GCS_JOB_DIR/export/JSON/`
    for `JSON`. See other formats `CSV` and `TFRECORD`.
- ```
- export MODEL_BINARIES=$GCS_JOB_DIR/export/JSON/
- ```
+ 
+```
+export MODEL_BINARIES=$GCS_JOB_DIR/export/CSV/
+```
 
 ```
-gcloud ml-engine versions create v1 --model census --origin $MODEL_BINARIES --runtime-version 1.4
+gcloud ml-engine versions create v1 --model census --origin $MODEL_BINARIES --runtime-version 1.10
+
 ```
 
-### (Optional) Inspect the model binaries with the SavedModel CLI
+(Optional) Inspect the model binaries with the SavedModel CLI
 TensorFlow ships with a CLI that allows you to inspect the signature of exported binary files. To do this run:
 
 ```
 SIGNATURE_DEF_KEY=`saved_model_cli show --dir $MODEL_BINARIES --tag serve | grep "SignatureDef key:" | awk 'BEGIN{FS="\""}{print $2}' | head -1`
 saved_model_cli show --dir $MODEL_BINARIES --tag serve --signature_def $SIGNATURE_DEF_KEY
 ```
 
-### Run Online Predictions
+* **Run Online Predictions**
```

Comment:
this is no census-specific.  consider factoring it out as a generic guide to online predictions?  such a guide must already exist on the doc site; are we considering simply linking to the doc site?

Reply:
Agree, my goal was that this README.md to be self-contained with mininum back and forth between different docs. In this case this is Census and Iris specific, other samples inference like Flowers may be different.